pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -73,6 +73,7 @@ type getconfigContext struct {
 	globalProfile            string
 	localProfile             string
 	localProfileTrigger      chan Notify
+	retryUpdateCounter       uint32
```

Comment:
When I read this I wondered which retryUpdateCounter it is - current? config? Turns out is it the parsed BaseOs which isn't clear. But shouldn't that be published as part of some new BaseOs publication which also includes the content tree UUID? This seems to be very partial

Reply:
OK, given that it will handle a new retryCounter, it does add some value and we can defer the switch from BaseOsConfig to BaseOs message in the API until later.